madmom/models

```diff
@@ -1 +1 @@
-Subproject commit c12fab69ae2ebc33965587867f5b230db60a1f08
+Subproject commit f80ae84e08df4cdb32347a8882300f26ba72985b
```

Comment:
Definitely get rid of this, since there are no new models this code relies on., right?